package.json

```diff
@@ -25,12 +25,14 @@
     "prepack": "yarn build"
   },
   "dependencies": {
-    "@jupiterone/integration-sdk": "^3.1.0",
+    "@jupiterone/integration-sdk-core": "^1.0.2",
     "@lifeomic/attempt": "^3.0.0",
     "base-64": "^0.1.0",
     "node-fetch": "^2.6.0"
   },
   "devDependencies": {
+    "@jupiterone/integration-sdk-cli": "^1.0.2",
+    "@jupiterone/integration-sdk-testing": "^1.0.2",
```

Comment:
I think the goal is to have `@jupiterone/integration-sdk-dev-tools` provide these, right @charlieduong94, along with all the changes to config files to use the shared stuff?